docs/using_eessi/setting_up_environment.md

```diff
@@ -1,6 +1,55 @@
 # Setting up your environment
 
-To set up the EESSI environment, simply run the command:
+To set up the EESSI environment, you can either load an EESSI module with Lmod or source a bash initialisation script.
+
+## Loading an EESSI module with Lmod
+
+There are a few different scenarios where you may want to set up the EESSI environment by loading an EESSI module with Lmod:
+
+   !!! note "Why do we recommend to unset `MODULEPATH`?"
+
+        Unsetting MODULEPATH may not be possible at some sites, and some compromise or workaround may
+        be necessary. The underlying reason to suggest this is that EESSI and your system are most likely based on
+        two different operating system distributions - EESSI uses Gentoo Prefix, your system almost certainly uses some other
+        distribution. If you can find a way to ensure that the _software stacks_ from your site and EESSI do not mix
+        (in particular when someone is building new software!), then this should be good enough.
+
+1.  You are already using Lmod with version >= 8.6
+
+    In this case, we _recommend_ unsetting the `MODULEPATH`, because EESSI is not designed to mix modules coming from EESSI and from your system.
+
+    ``` { .bash .copy }
+    unset MODULEPATH
+    export MODULEPATH=/cvmfs/software.eessi.io/init/modules
+    module load EESSI/2023.06
+    ```
+
+    :clap: Your environment is now set up, you are ready to start running software provided by EESSI!
+
+2.  You are using an Lmod with a version older than 8.6 or any other tool utilizing `MODULEPATH` (Tmod, etc.)
+
+    You should unset MODULEPATH to prevent Lmod from attempting to build a cache for your module tree (as this can be very slow if you have
```

Comment:
```suggestion
    You should unset `$MODULEPATH` to prevent Lmod from attempting to build a cache for your module tree (as this can be very slow if you have